scripts/version.mjs

```diff
@@ -0,0 +1,46 @@
+import { readFileSync, writeFileSync } from 'fs';
```

Comment:
Can we use https://github.com/antfu-collective/bumpp instead of using a handwritten script? Most of our plugin repos use `bumpp`